src/Sylius/Bundle/ApiBundle/composer.json

```
@@ -25,8 +25,8 @@
     "require": {
         "php": "^8.2",
         "doctrine/dbal": "^3.9",
-        "api-platform/doctrine-orm": "^4.1.7",
-        "api-platform/symfony": "^4.1.7",
+        "api-platform/doctrine-orm": "^4.2.0",
+        "api-platform/symfony": "^4.2.0",
```

We don’t specify the path version for other core dependencies (e.g., Symfony, Doctrine, etc.).

```suggestion
        "api-platform/doctrine-orm": "^4.2",
        "api-platform/symfony": "^4.2",
```

---

This bump should be reverted.

---

As above.